src/GitHubApps/src/DarcBot/AzureFunction.cs

```diff
@@ -0,0 +1,212 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Kusto.Ingest;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.DotNet.Kusto;
+using Microsoft.Extensions.Logging;
+using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace DarcBot
+{
+    public static class GitHubWebHook
+    {
+        private static ILogger _log;
+        private static Regex _darcBotIssueIdentifierRegex = new Regex(@"\[BuildId=(?<buildid>[^,]+),RecordId=(?<recordid>[^,]+),Index=(?<index>[^\]]+)\]", RegexOptions.Multiline | RegexOptions.IgnoreCase | RegexOptions.IgnorePatternWhitespace);
+        private static Regex _darcBotPropertyRegex = new Regex(@"\[(?<key>.+)=(?<value>[^\]]+)\]", RegexOptions.Multiline | RegexOptions.IgnoreCase | RegexOptions.IgnorePatternWhitespace);
+        private static readonly string docLink = "[DarcBot documentation](https://github.com/dotnet/arcade-services/tree/master/src/GitHubApps/src/DarcBot/Readme.md)";
+
+        [FunctionName("GitHubWebHook")]
```

Comment:
Is this another azure function on top of the one Jon deploys? Is there some way we can consolidate some of this stuff, so we don't have so many services we have to manage? I don't necessarily have a _good_ answer to this... I think we are going to need a team wide discussion around our plans here.

Reply:
@jonfortescue  can you point me to yours...?

Reply:
@jonfortescue , how does [this](https://github.com/dotnet/arcade-services/tree/master/src/RolloutScorer) deploy to Azure?  Or, is that not wired up yet?